### DISCUSSION TITLE
loadingProgression value of useUnityContext hook reports nothing for compressed builds

### DISCUSSION THREAD

**Discussion by mattsykesd2** [jeffreylanters/react-unity-webgl#454] [Help Wanted!]
### Please avoid duplicates

- [X] I checked [all open issues](https://github.com/jeffreylanters/react-unity-webgl/issues) and [discussions](https://github.com/jeffreylanters/react-unity-webgl/discussions) and none of them matched my problem.

### Language and Compiler

TypeScript's built-in Compiler

### What environment are you using?

Static File Serving

### When does your problem occur?

While the Unity App is loading

### What does your problem relate to?

The problem seems Module related

### React-Unity-WebGL Version

9.0.4

### React Version

16.14

### Unity Version

2022.1.11f1

### What happened?

The 'loadingProgression' value of the useUnityContext hook does not seem to work with gzip compressed unity builds.

My host works fine with non compressed builds and reports load progress as expected.

However, if I try to host a compressed build, loadingProgression will sit at 0 until the zip file is downloaded at which point it will jump to 90. It will load and run the build, it's just not reporting the load progress.

All server side content headers are set as required.

### Would you be interested in contributing a fix?

- [ ] yes, I would like to contribute a fix

**Comment by jeffreylanters** [comment]
Hi. I've searched online for similar issues and might have found a possible candidate which might cause this issue. I've included this fix in the latest release of version `9.1.0`. Could you try this version to see if this fixes your issue?

**Reply by mattsykesd2** [reply]
Hi, just tried with the updated version and still the same problem unfortunately
